src/zabapgit_unit_test.prog.abap

```diff
@@ -38,6 +38,11 @@ DEFINE _append_result.
   <result>-filename = &8.
 END-OF-DEFINITION.
 
+DEFINE _given_source_is.
```

Comment:
can we move this macro to a local method instead? The code will not look as pretty, but in this case it is easy to get rid of the macro, also chaining should mainly be used for declarations, https://help.sap.com/http.svc/rc/abapdocu_751_index_htm/7.51/en-US/abenchained_statements_guidl.htm